src/type/SaveInContainerOptions.ts

```diff
@@ -0,0 +1,38 @@
+//
+// Copyright 2022 Inrupt Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal in
+// the Software without restriction, including without limitation the rights to use,
+// copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+// Software, and to permit persons to whom the Software is furnished to do so,
+// subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+// PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+// HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+// SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+
+import { FetchOptions } from "./FetchOptions";
+/**
+ * @module interfaces
+ */
+
+/**
+ * Additional options for saving a dataset/resource in a container. Extends
+ * FetchOptions and includes:
+ * - `slugSuggestion`: A user suggested resource/container name. If a container
+ *   does not already have a resource/container in it with that name, it will
+ *   use the slug suggestion as the name.
+ *
+ * @since unreleased
+ */
+export interface SaveInContainerOptions extends FetchOptions {
```

Comment:
Can we keep this TODO:
```suggestion
// TODO: This should come from @inrupt/solid-client
export interface SaveInContainerOptions extends FetchOptions {
```